### ISSUE TITLE
No text in currently selected window in popup.

### ISSUE THREAD

**Issue by Mr-Crabman** [nsz32/docklike-plugin#77]
A screenshot of what I mean:

https://imgur.com/a/9laVMgQ

Whichever window I click on has the text vanish (but the text comes back to whichever other one I had selected previously). This has been an issue since I started using the plugin.

I'm using Xubuntu 20.04.1

**Comment by tomas-chrastina** [comment]
Not sure if it helps, but I found out something similar happening to me in some of Xfce themes.
So you can try if its theme related by changing your Appearance/Window Manager theme, restart panel.
Then if it's theme problem, I guess only some way of docklike theming would help.

**Comment by Mr-Crabman** [comment]
Changing the theme does seem to work; unfortunate that every theme except greybird seems to work fine.

> Then if it's theme problem, I guess only some way of docklike theming would help.

Could you rephrase? I don't understand what you're saying here.

**Comment by tomas-chrastina** [comment]
I meant, that if the theme is problem, then there is needed some change in docklike theme.

And it would be better to have some way to customize it by user, because ano other theme can be a problem.
Or do some automatic check of colors to use contrast colors for text (foreground) and bars (background).

**Comment by davekeogh** [comment]
Hi. Could someone confirm if this issue still persists with the latest version from git? I can figure out quite why this is happening, but I added a workaround that seems to work for the Greybird theme at least.

https://github.com/nsz32/docklike-plugin/commit/cb07cf4211b89c12375d8a911aec30287efe19d7

**Comment by SwampRabbit** [comment]
@davekeogh I will test to help, MX Linux uses our own versions of Greybird, but **I don't believe** it is still affecting me on other themes recently like it was before.  Give me a day to test and confirm.

**Comment by SwampRabbit** [comment]
@davekeogh I have tested with Greybird without issues, tested with some other themes (light and dark), but I believe this has been fixed pending someone else testing too.  But I'll keep testing either way.

**Comment by Mr-Crabman** [comment]
Having tested this now as well, it seems this issue has been fixed now.
